dev/DevBuildController.php

```diff
@@ -16,16 +16,15 @@ public function build($request) {
 			$da = DatabaseAdmin::create();
 			return $da->handleRequest($request, $this->model);
 		} else {
-			$renderer = DebugView::create();
-			$renderer->writeHeader();
-			$renderer->writeInfo("Environment Builder", Director::absoluteBaseURL());
-			echo "<div class=\"build\">";
+			if(Director::isDev() || Permission::check("ADMIN")) {
```

Comment:
I'm wondering why this check wasn't in the code in the first place. At what other point were non-admin users being rejected from this action on test/live?